Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -5376,6 +5608,488 @@ def update_vulnerability_exception_status_command(client: Client, vulnerability_
     )
 
 
+def create_tag_command(client: Client, name: str, type: str, color: str, ip_address_is: str | None = None,
+                       host_name_is: str | None = None, risk_score_higher_than: str | None = None,
+                       vulnerability_title_contains: str | None = None, site_id_in: str | None = None,
+                       site_name_in: str | None = None, query: str | None = None, match: str | None = None):
+    """
+    Create a tag.
+
+    Args:
+        client (Client): Client to use for API requests.
+        name (str): The tag name.
+        type (str): The tag type.
+        color (str): The tag color - relevant only for "Custom" type.
+        ip_address_is (str, optional): A specific IP address to search for.
+        host_name_is (str, optional): A specific host name to search for.
+        risk_score_higher_than (str, optional): A minimum risk score to use as a filter.
+        vulnerability_title_contains (str, optional): A string to search for in vulnerability titles.
+        site_id_in (str, optional): Site IDs to filter for. Can be a comma-separated list.
+        site_name_in (str, optional): Site names to filter for. Can be a comma-separated list.
+        query (str, optional): Additional queries to use as a filter, in the format: {field} {operator} {value}.
+                                Multiple queries can be specified, separated by a ";" separator.
+        match (str, optional): Operator to determine how to match filters.
+                                "All" requires all filters to match, "Any" requires only one filter to match.
+
+    Returns:
+        CommandResults: Results of the tag creation.
+    """
+    validate_input(type, VALID_TAG_TYPES, "type", True)
+    validate_input(color, VALID_TAG_COLORS, "color", False)
+
+    if type != "Custom" and color != "Default":
```

Comment:
change it to lower case.
```suggestion
    if type.lower() != "custom" and color.lower() != "default":
```